### DISCUSSION TITLE
What does the -f option do?

### DISCUSSION THREAD

**Discussion by pfofi** [trapexit/mergerfs#1356] [Q&A] [answered]
I was wondering what the `-f` option does in `mergerfs -f`. It is only mentioned as part of the [systemd mount example](https://github.com/trapexit/mergerfs?tab=readme-ov-file#systemd-mount) and not explained further.

I've noticed mergerfs won't stay alive if I start it via systemd without this option. Does it stand for 'foreground', so that mergerfs won't detach itself from the invoking process?

**Comment by trapexit** [comment, marked as answer]
Run `mergerfs --help` and it will explain it.

Yes, it is libfuse's standard "run in foreground" argument. By default it daemonizes.
